### PR TITLE
CFD-293 Homepage: Activity stream based on the filter (Logged-in users)

### PR DESCRIPTION
**Pull request by HelenaEksler**
Behat tests for homepage activity stream.

### CODE REVIEW

File: capacity4more/behat/features/homepage.feature [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  @javascript
+  Scenario: Change one group access to restricted.
+    Given I am logged in as user "admin"
+    When  I change access of group "Nobel prize" to Restricted with "gravity.com" restriction
+    Then  I should not see "error"
+
+  @javascript
+  Scenario: Anonymous user can see article activities and doesn't see filter.
+    Given I am an anonymous user
+    When  I visit ""
```
Comment (amitaibu) [inline capacity4more/behat/features/homepage.feature]: ``When  I visit "/"``

or maybe ``When I visit homepage``

File: capacity4more/behat/features/homepage.feature [hunk 2]
```diff
@@ -0,0 +1,62 @@
+Feature: Test homepage activity stream
+  In order to see recent content
+  As a group member and non-member and anonymous user
+  I need to be able to see an activity stream of recent operations
+
+  @javascript
+  Scenario: Change one group access to restricted.
```
Comment (zero2one) [inline capacity4more/behat/features/homepage.feature]: Should this test be located in the group tests?

Comment (HelenaEksler) [inline capacity4more/behat/features/homepage.feature]: In this case we probably will have to change many other tests. So I think we can change access,  check what we want and change it back.

File: capacity4more/behat/features/homepage.feature [hunk 3]
```diff
@@ -0,0 +1,62 @@
+Feature: Test homepage activity stream
```
Comment (zero2one) [inline capacity4more/behat/features/homepage.feature]: There are 2 features for homepage:

* homepage_anonymous.feature
* homepage_logged_in.feature

Comment (HelenaEksler) [inline capacity4more/behat/features/homepage.feature]: I'll merge them in one.

**Comment by HelenaEksler** [comment]
@zero2one Could you review this please and merge if it's ok?

File: capacity4more/behat/features/homepage.feature [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    And   I should not see "Nobel Prize" in the "div.activity-stream" element
+    And   I should see "posted an Article" in the "div.activity-stream" element
+
+  @javascript
+  Scenario: Logged in user not member should see only activities from groups of
+  interests when filter is set to My interests
+    Given I am logged in as user "president"
+    When  I visit the site homepage
+    And   I select the radio button "My interests" with the id "edit-homepage-filter-interests"
+    And   I wait
```
Comment (amitaibu) [inline capacity4more/behat/features/homepage.feature]: that's a bit ugly ;) We should have ``I should wait not to see ...``

**Pull request merged**